pkg/configtx/msp_test.go

```diff
@@ -419,7 +419,7 @@ func TestMSPToProtoFailure(t *testing.T) {
 	fabricMSPConfig.SigningIdentity.PrivateSigner.KeyMaterial = &ecdsa.PrivateKey{}
 
 	fabricMSPConfigProto, err := fabricMSPConfig.toProto()
-	gt.Expect(err).To(MatchError("pem encode PKCS#8 private key: marshaling PKCS#8 private key: x509: unknown curve while marshaling to PKCS#8"))
+	gt.Expect(err).To(MatchError("pem encode PKCS#8 private key: marshaling PKCS#8 private key: x509: unknown curve while marshalling to PKCS#8"))
```

Comment:
Accidental spelling change here? I was checking the cause of your CI failure just to be sure my recently merged PR didn't mess things up somehow but this is the cause.

Reply:
I thought I saw this fail locally and changed the spelling. I'll revert this change.